cns/ipampoolmonitor/metrics.go

```diff
@@ -0,0 +1,70 @@
+package ipampoolmonitor
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	ipamAllocatedIPCount = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "ipam_allocated_ip",
+			Help: "Allocated IP count.",
+		},
+	)
+	ipamAvailableIPCount = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "ipam_available_ip",
+			Help: "Available IP count.",
+		},
+	)
+	ipamBatchSize = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "ipam_batch",
```

Comment:
thoughts on `size` as a suffix? Same for `pool` below.

Reply:
works for me